.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2.1
+
+var_1: &docker_image circleci/node:10.13.0-browsers
+var_2: &cache_key raiden-explorer-{{ .Branch }}-{{ checksum "package-lock.json" }}
+
+anchor_1: &root_package_lock_key
+  key: *cache_key
+
+anchor_2: &attach_options
+  at: .
+
+executors:
+  base-executor:
+    working_directory: ~/src
+    docker:
+      - image: *docker_image
+
+jobs:
+  install:
+    executor: base-executor
+    working_directory: ~/src/frontend
+    steps:
+      - checkout:
+          path: ~/src
+      - restore_cache: *root_package_lock_key
+      - run: npm ci
+      - save_cache:
+          <<: *root_package_lock_key
+          paths:
+            - "node_modules"
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./*
+
+  lint:
+    executor: base-executor
+    working_directory: ~/src/frontent
```

Comment:
This should be fronten*d*, wonder hwo this works ;)

Reply:
OK let me see if it works after removing them completely then :+1:

Reply:
It is probably because this is also setup in the executor

Reply:
but it seems that the install part still needs the working_directory specified or it will fail.